wiki/osu!tourney/kr.md

```diff
@@ -0,0 +1,353 @@
+
+경기가 종료되면 점수가 높은 팀에게 별 하나가 채워지며 팀 점수(별)를 증가됩니다. 또한 팀점수를 수동으로 변경할 수 있도록 하여. **마우스 오른쪽키와 왼쪽키를 클릭하여 별의 개수를 수동으로 변경 할 수 있습니다.**.
+
+![osu!tourney with activated annotation](Osutourneywarmup.png "osu!tourney with activated annotation")
+
+메시지를 표시하려면 주석단추를 클릭하여 설정할 수 있습니다. 활성화되면 제어판의 해당 텍스트 상자에 있는 메시지가 화면 상단에 표시되고 팀 점수는 beatmap이 완료된 후에도 변경되지 않습니다.
+
+아래쪽 컨트롤 패널의 각 버튼의 기능은 아래와 같습니다.:
+
+-   **Sync music:** 음악이 동기화되지 않는다면 버튼을 누릅니다. osu!tourney은 음악을 다시 동기화하려고 시도합니다.
+-   **Toggle annotation:** 주석을 끄거나 켭니다.(자동화된 팀 점수 계산에 영향을 미칩니다)
+-   **Panic:** 프로그램이 잘못되어 창이 닫히거나 윈도우 충돌이 발생 할 경우 이를 누릅니다. 그러면 모든 스트림 윈도우가 열립니다.
+-   **Exit:** osu!tourney를 종료합니다.
+
+아래에 있는 제어판의 각 텍스트 상자의 가능은 아래에 설명되어 있습니다.
+
+-   **주석 텍스트:** 주석에 표시할 텍스트입니다.
+-   **최대 승리수:** 각각의 팀이 이겨야 할 별의 개수를 지정합니다.
+
+------------------------------------------------------------------------
+
+Multiplayer 세션 생성 및 취급
+=========================================
+
+매칭설정
+--------------
+
+osu!tourney을 사용하여 멀티 방을 원활하게 상호작용을 하기 위해서는 몇가지 사항에 유의해야 합니다.
+
+먼저 멀티 방을 생성하고 제어판에 나열된 템플릿을 바탕으로 이름을 지정해야 합니다. 이 템플릿은 다음 구성 요소로 구성됩니다. 
+
+`Your_acronym_in_tournament.cfg: (팀 이름 1) vs (팀 이름 2)`
+
+`Your_acronym_in_tournament.cfg` 설치 중에 tournaments.cfg에서 정의한 값으로 대체됩니다. 예를 들어, 이 값은 **[Setup](Osu!tourney#Setup "wikilink")**. "Test Tourney"로 정의되었습니다. 여기 다음 예제에서는 템플릿을 사용합니다.
+
+`Test Tourney: (팀 이름 1) vs (팀 이름 2)`
+
+팀 이름 1과 팀 이름 2를 임의로 할당할 수 있지만, 괄호를 묶어 두면 됩니다. **괄호를 묶어 두면 됩니다 () **.
+
+IRC Commands (optional)
+-----------------------
+
+이것을 사용하려면 모든 토너먼트 스텝과 도우미가 IRC를 통해 Bancho (반쵸)에 연결되어 있고, [토너먼트 스텝](https://osu.ppy.sh/g/26) 일원이 되어야 됩니다.
+
+다음 명령어는 모두 IRC 클라이언트를 통해 실행해야합니다. (인기 있는 선택사항은 mIRC, HexChat, HydraIRC입니다).
+
+Bancho (반쵸)에게 개인 메시지로 !mp help를 보내면 모든 명령 목록을 볼 수 있습니다. 멀티방을 생성하려면 Bancho (반쵸)에게 메시지를 보내십시오. `!mp make Test Tourney: (팀 이름 1) vs (팀 이름 2)`.
+
+이 방은 모든 플레이어들이 방을 나갔을 때 닫히지 않는 점에서 특별하며 그것은 본질적으로 방에 들어가는 플레이어들부터 보호됩니다. 모든 토너먼트 스텝 IRC 클라이언트는 자동적으로 새로 만든 멀티방의 채팅 채널에 자동적으로 참여합니다. 모든 멀티방의 명령은 오직 채팅 채널에서만 작성되어야 합니다.
+
+경기가 끝나면, `!mp close` 명령을 보내 방을 닫아야 합니다. 
+
+사용 가능한 관리 명령 목록은 다음과 같습니다. 단 대괄호 []로 묶인 속성은 선택 사항입니다.
+
+-   `!mp lock` - 플레이어가 팀과 슬롯을 변경할 수 없도록 방을 잠급니다.
+-   `!mp unlock` - 위의 내용을 반대로 바꿉니다.
+-   `!mp size` <size> -방에서 사용 가능한 슬롯(1-16)의 양을 설정합니다.
+-   `!mp set` <teammode> `[<scoremode>] [<size>]` - 다양한 방 속성을 설정합니다. 
+
+    `teammode` - 0: 개인전, 1: 태그 개인전, 2: 팀전, 3: 태그 팀전 Vs
+
+    `scoremode` - 0: 스코어, 1: 정확도, 2: 콤보
+
+-   `!mp move` <username> `[<slot>]` - 플레이어를 방으로 이동시켜 지정된 슬롯에 넣습니다. 플레이어가 이미 방에 있으면 플레이어는 지정된 슬롯으로 이동합니다.
+-   `!mp host` <username> - 플레이어에게 방장을 전달합니다.
+-   `!mp clearhost` - 방장을 지웁니다.
+-   `!mp settings` - 전체 세부 정보를 표시합니다.
+-   `!mp start [<time>]` - 설정된 시간 (초) 후에 또는 시간이 없는 경우에는 즉시 시작합니다.
+-   `!mp abort` - 경기를 중단합니다.
+-   `!mp kick` <username> - 플레이어를 추방시킵니다.
+-   `!mp team` <username> <colour> - 플레이어를 지정된 팀으로 이동시킵니다.
+
+    `colour` - red, blue
+
+-   `!mp map` <mapid> `[<playmode>]` - 방의 비트맵과 플레이모드를 변경합니다.
+
+    `playmode` - 0: osu!, 1: Taiko, 2: Catch The Beat, 3: osu!Mania
```

Comment:
New naming convention used. The one provided was based on the old name version.

Just replace the 

```Markdown
`playmode` - 0: osu!, 1: Taiko, 2: Catch The Beat, 3: osu!Mania
```` 

with the line provided below:

```Markdown
`playmode` - 0: osu!, 1: osu!taiko, 2: osu!catch, 3: osu!mania
```

No need to translate.